src/main/java/org/assertj/core/api/AbstractObjectArrayAssert.java

```diff
@@ -985,6 +985,32 @@ public SELF hasOnlyElementsOfTypes(Class<?>... types) {
     return myself;
   }
 
+  /**
+   * Verifies that all elements of actual group are instances of given types{@code Iterable}, in given order.
```

Comment:
reword to:
```
Verifies that the elements are of the given types (in the given order).
```
and use the same wording in `AbstractIterableAssert`, `AtomicReferenceArrayAssert` and `ObjectEnumerableAssert`.